testapi.pm

```diff
@@ -1810,7 +1810,7 @@ sub upload_logs {
 
     bmwqemu::log_call(file => $file);
     my $basename = basename($file);
-    my $upname   = ref($autotest::current_test) . '-' . $basename;
+    my $upname   = $args{upname} || ref($autotest::current_test) . '-' . $basename;
```

Comment:
better use `//` instead of `||` as you want to check if the parameter is appied at all

Reply:
just discussed with @okurz that suggestion is not relevant.